pkgs/development/python-modules/pysol_cards/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchPypi, python }:
+
+python.pkgs.buildPythonPackage rec {
+  pname   = "pysol_cards";
+  version = "0.8.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0w0waj7r1kqlpic6g3pyg4208i393gc0zxb6n6k0vqhm6nivdcs5";
+  };
+
+  propagatedBuildInputs = with python.pkgs; [ pbr random2 six ];
+
+  ## Need to fix test deps, relies on stestr and a few other packages that aren't available on nixpkgs
+  #checkInputs = with python.pkgs; [ pbr testtools stestr ];
+  doCheck = false;
```

I believe stestr suites are also able to be ran with pytest.
```suggestion
  ## Need to fix test deps, relies on stestr and a few other packages that aren't available on nixpkgs
  checkInputs = [ testtools pytest ];
  doCheck = "pytest";
```